src/coreclr-debug/activate.ts

```diff
@@ -22,7 +22,9 @@ export async function activate(thisExtension: vscode.Extension<CSharpExtensionEx
 
     if (!CoreClrDebugUtil.existsSync(_debugUtil.debugAdapterDir())) {
         let isValidArchitecture: boolean = await checkIsValidArchitecture(platformInformation, eventStream);
-        if (!isValidArchitecture) {
+        // If this is a valid architecture, we should have had a debugger, so warn if we didn't, otherwise
+        // a warning was already issued, so do nothing.
+        if (isValidArchitecture) {
```

Comment:
Shouldn't this still be `!isValidArchitecture`? `checkIsValidArchitecture` returns true if we support it, else `false`.

Reply:
No. The goal of this code was to add a warning if this is a valid architecture, but we somehow didn't download a debugger.

Reply:
This was broken by Austen's ARM64 PR, and we didn't notice because the check is a bit counter intuiative (which is why I added a comment).